From 623b4ae1d94648bc034dd96523e6a6ae1a506b44 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi@gmx.de>
Date: Fri, 6 Apr 2018 22:33:45 +0200
Subject: [PATCH] Bump Security Patch Level to 2018-04-01

Implemented patches
===================
CVE-2017-13274	A-71360761	High		Change 211308
CVE-2017-13276	A-70637599	Critical	Change 211309
CVE-2017-13277	A-72165027	Critical	Change 211310
CVE-2017-13278	A-70546581	High		Change 211347
CVE-2017-13279	A-68399439	High		Change 211348
CVE-2017-13280	A-71361451	High		Change 211349
CVE-2017-13267	A-69479009	Critical	Change 211350
CVE-2017-13284	A-70808273	Critical	Change 211351
CVE-2017-13287	A-71714464	High		Change 211352
CVE-2017-13289	A-70398564	High		Change 211353
CVE-2017-13290	A-69384124	High		Change 211354
CVE-2017-13294	A-71814449 [1]	Moderate	Change 211364
CVE-2017-13294  A-71814449 [2]	Moderate	Change 211365
CVE-2017-13295	A-62537081	Moderate	Change 211368
CVE-2017-13296	A-70897454	High		Change 211355
CVE-2017-13297	A-71766721	High		Change 211357
CVE-2017-13298	A-72117051	High		Change 211358
CVE-2017-13299	A-70897394	High		Change 211356

Skipped patches
===============
CVE-2017-13285  A-69177126      High		Can't fork svox due to DMCA
CVE-2017-13300  A-71567394*     High		Not publicly available

Change-Id: I8f2f06b674b3eb63648765e272d0623c35c04d60
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 21852e2..74a938a 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -105,7 +105,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
   #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
   #
   #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-  PLATFORM_SECURITY_PATCH := 2018-03-01
+  PLATFORM_SECURITY_PATCH := 2018-04-01
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.7.4

